css/style.css

```diff
@@ -327,3 +328,75 @@ div#tadd select { width: 220px }
 @media only screen and (max-width: 729px) {
   .mobile-search { display: none; }
 }
+
+/* X-Small devices (portrait phones, less than 576px) */
+/* No media query for `xs` since this is the default in Bootstrap */
+/* Break point settings borrowed from Bootstrap framework */
```

Comment:
What do you think about adding Bootstrap to ruTorrent? This is the preferred solution over importing settings to `style.css`. Modular programming practices are very important for this project.

Reply:
Yeah, I think it's a good idea. I thought jQuery was the only dependency library that ruTorrent requires, making it a light-weighted web app. I was not sure about bringing another dependency into ruTorrent, so I reinvented the wheel. (Maybe just a tiny bit of the wheel, haha).

Reply:
Or maybe we could consider TailwindCSS, as it focuses on utility classes for highly customizable designs. We already have a very mature user interface already. We just need a more tailored polish for both all kinds of devices.

Reply:
Should we use Bootstrap 4, which has reached end of life, or the latest version of Bootstrap?

Reply:
Good catch, let's use the latest version of Bootstrap. It will need to be distributed with the project like jQuery.

Reply:
Yeah, sure. I've been doing tests with the latest Bootstrap version 5.3.3. 

At first, I tried to link the entire Bootstrap library, and it seemed to interfere with the custom CSS classes defined in `/css/style.css`. The left side-panel became very crowded, and the tables became vertically unaligned between table heads and table bodies. The huge leap forward requires a lot of fine-tuning of the entire ruTorrent project.

So I tried to minimize the impact by using `bootstrap-grid.min.css` instead, as we are mainly working on importing the responsive grid system from Bootstrap. The effect is smooth. The modification on the add torrent dialog is mostly done. I will push a patch commit to this pull request in the coming days after doing more tests.